_articles/legal.md

```diff
@@ -54,21 +54,23 @@ When you create a new project on GitHub, you'll be [asked to add a license](http
 
 ## Which open source license is appropriate for my project?
 
-If you're starting from a blank slate, it's hard to go wrong with the [MIT License](https://choosealicense.com/licenses/mit/). It's short, very easy to understand, and allows anyone to do anything so long as they keep a copy of the license, including your copyright notice. You'll be able to release the project under a different license if you ever need to.
```

Comment:
As I understand it, the goal of this first paragraph is to *quickly* give readers the name of license they can use and move on with their lives if they are tired of thinking about legal stuff.

This rephrasing puts the name of MIT in the first line instead of the second.